doc/external-libs/cuda.html

```diff
@@ -15,6 +15,10 @@ <h1>Installing deal.II with CUDA</h1>
     <p>
       To compile and run CUDA code, you need to use an external Kokkos
       installation that has been configured with CUDA support.
+      Note that deal.II requires a compiler that is compatible with the
+      backends enabled in Kokkos. In particular, you need to use Kokkos'
+      <code>nvcc_wrapper</code> or <code>clang++</code> when enabling the
+      Cuda backend.
```

Comment:
```suggestion
      backends enabled in Kokkos. In particular, you need to explicitly define Kokkos'
      <code>nvcc_wrapper</code> or <code>clang++</code> as `CMAKE_CXX_COMPILER` when enabling the
      Cuda backend.
```

Not a strong opinion, though, just what I got from the discussion